Day 4 tasks/tasks4.js

```diff
@@ -0,0 +1,198 @@
+    console.log("B");
+} else if (score >= 60 && score < 70) {
+    console.log("C");
+} else if (score >= 50 && score < 60) {
+    console.log("D");
+} else if (score >= 0 && score < 50) {
+    console.log("F");
+} else {
+    console.log("Invalid score.");
+}
+
+//2.Check if the season is Autumn, Winter, Spring or Summer. If the user input is :
+/*September, October or November, the season is Autumn.
+December, January or February, the season is Winter.
+March, April or May, the season is Spring
+June, July or August, the season is Summer*/
+
+let month = prompt("Enter a month:");
+
+switch(month.toLowerCase()) {
+    case 'september':
+    case 'october':
+    case 'november':
+        console.log("The season is Autumn.");
+        break;
+    case 'december':
+    case 'january':
+    case 'february':
+        console.log("The season is Winter.");
+        break;
+    case 'march':
+    case 'april':
+    case 'may':
+        console.log("The season is Spring.");
+        break;
+    case 'june':
+    case 'july':
+    case 'august':
+        console.log("The season is Summer.");
+        break;
+    default:
+        console.log("Invalid month.");
+}
+
+//3. Check if a day is weekend day or a working day. Your script will take day as an input.
+    /*What is the day  today? Saturday
+    Saturday is a weekend.
+
+    What is the day today? saturDaY
+    Saturday is a weekend.
+
+    What is the day today? Friday
+    Friday is a working day.
+
+    What is the day today? FrIDAy
+    Friday is a working day.*/
+
+    let day = prompt("What is the day today?");
+
+    switch(day.toLocaleLowerCase()){
+        case 'saturday':
+        case 'sunday':
+            console.log(`${day}is weekend`);
+            break;
+        case 'monday':
+        case 'tuesday':
+        case 'wednesday':
+        case 'thursday':
+        case 'friday':
+            console.log(`${day}is a working dey`);
+            break;
+        default:
+            console.log('Invalid day');
+    }
+
+
+//Exercises: Level 3
+/*Write a program which tells the number of days in a month.
+  Enter a month: January
+  January has 31 days.
+
+  Enter a month: JANUARY
+  January has 31 day
+
+  Enter a month: February
+  February has 28 days.
+
+  Enter a month: FEbruary
+  February has 28 days.*/
+
+  let monthDays = prompt("Enter a month:");
+
+switch(monthDays.toLowerCase()) {
+    case 'january':
+    case 'march':
+    case 'may':
+    case 'july':
+    case 'august':
+    case 'october':
+    case 'december':
+        console.log(`${monthDays} has 31 days.`);
+        break;
+    case 'february':
+        console.log(`${monthDays} has 28 days.`);
+        break;
+    case 'april':
+    case 'june':
+    case 'september':
+    case 'november':
+        console.log(`${monthDays} has 30 days.`);
+        break;
+    default:
+        console.log("Invalid month.");
+}
+
+//Write a program which tells the number of days in a month, now consider leap year.
```

Comment:
Are you going to do this task?

Reply:
They were some difficult for me. I'm going to do them later